index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MdSaib</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="portHeader">
+    <a href="#" class="msLogo"> M S</a>
+    <nav id="menuBar" onclick="toggle()">
+      <img id="union" src="icon/union.svg" alt="menu">
+   </nav>
+  </header>
+
+<section class="homebody">
+  <article class="headline">
+    <div class="text1">
+      <h3>I’m Md Saib Glad to see you!</h3>
+    </div>
+    
+    <p id="text2">
+      I’m a software developer! I can help you build a product , feature or website
+      Look through some of my work and experience! If you like what you see and 
+      have a project you need coded, don’t hestiate to contact me.
+    </p>
+        
+    <a class="text3" href="#">LET’S CONNECT</a>
```

Comment:
- [ ] You have implemented this link in a good way. Let's make it better by removing the underline to improve the UI design.
<img width="233" alt="image" src="https://user-images.githubusercontent.com/77190203/149320199-7c4de90a-ae3a-4cf5-bb75-ea1308d139a3.png">